src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -138,5 +138,55 @@ public void validParameters_for_userTookQuiz() {
         Boolean actual = timing_properties_test.userTookQuiz(user_quiz_time, game_quiz_time);
         Assert.assertEquals(false, actual);
     }
+
+    @Test
+    //Checks if the timestamp given is a null value
+    public void nullParameter_for_newDayNewQuiz(){
+        QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
+ 
+        Boolean actual = timing_properties_test.newDayNewQuiz(null);
+        Assert.assertEquals(null, actual);
+    }
  
+    @Test 
+    //Checks if newDayNewQuiz works especially with being dependent on the getTimestampProperty
+    public void validParameter_for_newDayNewQuiz() {
+        QuizTimingPropertiesUtils timing_properties_test = new QuizTimingPropertiesUtils();
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I think that datastore and the timing properties test can be declared once for all the tests, its only instances of the class you are testing that would have to be declared in each separate test

Reply:
Devon suggested I create them for each test which is why I don't have them as global variables for the whole class

Reply:
ok cool thats fine then